src/ansys/platform/instancemanagement/instance.py

```diff
@@ -284,7 +285,11 @@ def build_grpc_channel(self, service_name: str = "grpc", **kwargs):
         if not service:
             raise UnsupportedServiceError(self.name, service_name)
 
-        return service._build_grpc_channel(**kwargs)
+        configuration = Configuration.from_environment()
+        if configuration.tls:
```

Comment:
I think the `tls` check should be done within the `Service` class, to keep together the check that `tls` is enabled, and reading the `tls`-specific part of the configuration. In the end, here the `Instance` class does not need to know if TLS is enabled or not when calling this method